vic/drivers/classic/src/vic_classic.c

```diff
@@ -247,6 +248,13 @@ main(int   argc,
             ******************************************/
 
             for (rec = startrec; rec < global_param.nrecs; rec++) {
+
+                // Set global reference string (for debugging inside vic_run)
+                sprint_dmy(dmy_str, &(dmy[rec]));
```

Comment:
Why do we always do this rather than only when we error? Presumably because the info is not available when we error?